subsys/logging/log_rpc_group.h

```diff
@@ -40,6 +41,8 @@ enum log_rpc_cmd_backend {
 	LOG_RPC_CMD_SET_HISTORY_LEVEL = 1,
 	LOG_RPC_CMD_FETCH_HISTORY = 2,
 	LOG_RPC_CMD_GET_CRASH_LOG = 3,
+	LOG_RPC_CMD_GET_BUFFER_USAGE_THRESHOLD = 4,
```

Comment:
ditto: can these be named like `BUFFER_HISTORY_THRESHOLD` to make it consistent with `LOG_RPC_EVT_HISTORY_THRESHOLD_REACHED`?